src/component/fieldtext/InputField.js

```diff
@@ -1,14 +1,26 @@
 import { ErrorMessage, Field } from "formik"
 
-import React from 'react'
+import React from "react"
 
 const InputField = ({ name, type, label, handleChangeInput }) => {
+  const handleChangeFc = (e, handleChange) => {
+    handleChangeInput(e, handleChange)
+  }
   return (
     <Field name={name}>
       {({ field, form: { handleChange } }) => (
         <div>
           <label htmlFor={name}>{label}</label>
-          <input type={type} {...field} onChange={(e) => handleChangeInput(e, handleChange)} id={name}></input>
+          <input
+            type={type}
+            {...field}
+            onChange={(e) => {
+              handleChangeInput
+                ? handleChangeFc(e, handleChange)
+                : handleChange(e)
+            }}
```

Comment:
I'm not sure the logic u are using here.

Reply:
why are you not sure? tell me more detail please

Reply:
the logic u are using too complicated for other to understand